app/views/root/index.html.erb

```diff
@@ -1,12 +1,19 @@
-<% if current_user %>
```

Comment:
Can we safely assume we have a `current_user` available now? Might be worth remarking on in the commit message for the button addition